sklearn/ensemble/_stacking.py

```diff
@@ -585,6 +585,25 @@ def _validate_final_estimator(self):
                 )
             )
 
```

Comment:
We should add more information in the documentation.

For the `estimators` docstring, we can add:

```
The type of estimator is generally expected to be a classifier. However, one
can pass a regressor for some use case (e.g. ordinal regression) but the
target `y` is expected to be encoded as numerical.
```

Reply:
We can add a "Notes" section as well.